deployment/7_upgrade_1_6.js

```diff
@@ -12,11 +12,12 @@ const TokenPriceProvider = require("../build/TokenPriceProvider");
 const MakerRegistry = require("../build/MakerRegistry");
 const ScdMcdMigration = require("../build/ScdMcdMigration");
 const MakerV2Manager = require("../build/MakerV2Manager");
+const TransferManager = require("../build/TransferManager");
 
 const utils = require("../utils/utilities.js");
 
 const TARGET_VERSION = "1.6.0";
-const MODULES_TO_ENABLE = ["ApprovedTransfer", "RecoveryManager", "MakerV2Manager"];
+const MODULES_TO_ENABLE = ["ApprovedTransfer", "RecoveryManager", "MakerV2Manager", "TransferManager"];
 const MODULES_TO_DISABLE = ["UniswapManager"];
 
 const BACKWARD_COMPATIBILITY = 3;
```

Comment:
Didn't we say we would set `BACKWARD_COMPATIBILITY` to `1` and remove the part of the script where we create a  `LegacyUpgrader`? Or are we addressing this in a separate PR?

Reply:
There's a separate PR for it https://github.com/argentlabs/argent-contracts/pull/91 as I wanted to double check the clients can support the upgrade through 1.4 -> 1.6